Upstream-Status: Pending

This patch fixes following kind of issues with automake 1.12

| configure.in:51: error: automatic de-ANSI-fication support has been removed

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/03

Index: libcroco-0.6.3/configure.in
===================================================================
--- libcroco-0.6.3.orig/configure.in
+++ libcroco-0.6.3/configure.in
@@ -47,8 +47,6 @@ AC_PROG_CC
 AC_PROG_INSTALL
 AC_PROG_CPP
 
-dnl Make sure we have an ANSI compiler
-AM_C_PROTOTYPES
 test "x$U" != "x" && AC_MSG_ERROR(Compiler not ANSI compliant)
 
 dnl Checks for libraries.
